src/gui/entry/EditEntryWidgetSSHAgent.ui

```diff
@@ -37,6 +37,9 @@
      </item>
      <item>
       <widget class="QSpinBox" name="lifetimeSpinBox">
+       <property name="accessibleName">
+        <string>Remove key from agent seconds</string>
```

Comment:
This shortened sentence construction has been used before, but I would perhaps insert "after" between "agent" and "seconds". A bit confusing otherwise.